maas-api/README.md

```
@@ -210,7 +210,26 @@ curl -sSk \
 
 > [!NOTE]
-> API keys are stored in a SQLite database (`/data/maas.db` in the container) with metadata including creation date, expiration date, and status. They can be listed and inspected individually. To revoke tokens, use `DELETE /v1/tokens` which revokes all tokens (ephemeral and API keys) by recreating the Service Account and marking API key metadata as expired.
+> API keys are stored in the configured database (see [Storage Configuration](#storage-configuration)) with metadata including creation date, expiration date, and status. They can be listed and inspected individually. To revoke tokens, use `DELETE /v1/tokens` which revokes all tokens (ephemeral and API keys) by recreating the Service Account and marking API key metadata as expired.
+
+### Storage Configuration
+
+By default, maas-api uses **in-memory SQLite** (data lost on restart). For persistent storage, use one of the [deployment examples](../deployment/examples/README.md):
+
+| Mode | Use Case | Deploy Command |
+|------|----------|----------------|
+| **In-memory** (default) | Development/testing 
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)